samples/guide/src/main/java/okhttp3/recipes/PostFile.java

```diff
@@ -30,7 +31,7 @@
   private final OkHttpClient client = new OkHttpClient();
 
   public void run() throws Exception {
-    File file = new File("README.md");
+    File file = new File(SamplesKt.getPrefix() + "README.md");
```

Comment:
I don't really like this. The goal of these samples is to stand alone.

Any alternatives?

Reply:
removed with body doubles